index.js

```diff
@@ -980,6 +980,54 @@
   //. ```
   S.init = S.slice(0, -1);
 
+  //# take :: Number -> [a] | String -> Maybe [a] | Maybe String
+  //.
+  //. Returns Just a list containing the first n elements of the given
+  //. list or Just the first n characters of a given string. If n exceeds
+  //. the length of the list or string, returns Nothing.
+  //.
+  //. ```javascript
+  //. > S.take(2, ["a", "b", "c", "d", "e"])
+  //. Just(["a", "b"])
+  //.
+  //. > S.take(4, 'abcdefg')
+  //. Just("abcd")
+  //.
+  //. > S.take(2, [])
+  //. Nothing()
+  //.
+  //. > S.take(2, '')
+  //. Nothing()
+  //.
+  //. > S.take(4, ['a', 'b', 'c'])
+  //. Nothing()
```

Comment:
This example is sufficient to demonstrate the failure case. Let's remove the `[]` and `''` examples in the interest of brevity.